tests/Controllers/Validations/TimeEntryControllerValidationTest.php

```diff
@@ -172,6 +185,26 @@ public function test_update_time_entry_fails_when_duration_is_less_than_1_minute
         $this->assertEquals('Minimum Entry time should be 1 minute.', $response->exception->getMessage());
     }
 
+    /** @test */
+    public function test_update_time_entry_fail_with_duplicate_time_entry()
+    {
+        $firstEntry = factory(TimeEntry::class)->create(['user_id' => $this->defaultUserId]);
+
+        $endTime = date('Y-m-d h:i:s', strtotime($firstEntry->end_time.'+1 hours'));
+        $secondEntry = factory(TimeEntry::class)->create([
+            'end_time' => $endTime,
```

Comment:
this is also bug, here time is overlapping and its running without failing, put validations for time overlapping.